### DISCUSSION TITLE
SonarQube setup on Azure App Service

### DISCUSSION THREAD

**Discussion by giscus** [rajbos/rajbos.github.io#48] [Comments]
# SonarQube setup on Azure App Service

SonarQube setup on Azure App Service

https://devopsjournal.io/blog/2019/04/29/SonarQube-Azure-Default-Settings

**Comment by gitricko** [comment]
You might wanna try sonarless which automates this in just a few steps.




Install CLI: `curl -s "https://raw.githubusercontent.com/gitricko/sonarless/main/install.sh" | bash`




Scan current directory: `sonarless scan`




Get results: `sonarless results`




Check out the automation

https://github.com/gitricko/sonarless




It even have a GitHub Action for GITHUB CI. The nice thing about this is that you don't need a host sonarqube
